modules/upi-managing-existing-hosts-in-a-upi-cluster.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc
+:_content-type: PROCEDURE
+[id="upi-managing-existing-hosts-in-a-upi-cluster_{context}"]
+= Optional: Managing existing hosts in a user-provisioned cluster by using the BMO
+
+You can optionally use the Bare Metal Operator (BMO) to manage existing bare-metal controller hosts in a user-provisioned cluster by creating a `BareMetalHost` object for the existing host. 
+It is not a requirement to manage existing user-provisioned hosts, however, you can enroll them as externally-provisioned hosts for inventory purposes.
+
+[IMPORTANT]
+====
+To manage existing hosts by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `true` to prevent the BMO from re-provisioning the host.
+====
+
+.Prerequisites
+
+* You created a user-provisioned bare-metal cluster.
+* You have baseboard management controller (BMC) access to the hosts.
+* You deployed a provisioning service in the cluster by creating a `Provisioning` CR.
+
+.Procedure
+
+. Create the `Secret` CR and the `BareMetalHost` CR.
+
+.. Save the following YAML in the `controller.yaml` file:
```

Comment:
Is step `a` part of step 1 or should it be step 2?

Reply:
It's part of step one.